hirundo/dataset_optimization.py

```diff
@@ -833,6 +830,7 @@ class DataOptimizationDatasetOut(BaseModel):
 class DataOptimizationRunOut(BaseModel):
     id: int
     name: str
+    dataset_id: int
```

Comment:
Please explain why we need this here all of a sudden?

Reply:
We don't. I just noticed it while changing other things. It was in the server response before anyway